.github/workflows/ci.yaml

```diff
@@ -87,8 +87,8 @@ jobs:
     needs: [artifacts]
     strategy:
       matrix:
-        k8s_version: ["v1.25.8", "v1.24.12", "v1.23.17"]
-        vault_version: ["1.13.1", "1.12.5", "1.11.9", "1.10.11"]
+        k8s_version: ["v1.24.13", "v1.25.9", "v1.26.4" ]
```

Comment:
I believe the latest kind release supports 1.27 as well.

Reply:
Thats right, but afaik we agreed on that the in case of k8s, we support 3 versions until the latest-1 version.

Reply:
I'm a but confused: why wouldn't we want to support the latest version? Unless of course there is some sort of incompatibility that keeps us from supporting it (and requires a breaking change to do so).

Reply:
I don't know, but I agree that it wouldn't hurt to add the latest version to this matrix 🙂 Should we drop 1.24 or let this be a 4 element list?

Reply:
Personally, I'd keep as much versions as we can.